src/Ticket.php

```diff
@@ -5447,6 +5450,8 @@ public static function cronPurgeTicket(CronTask $task)
             ]
         );
 
+        $max = $task->fields['param'] ?? 0;
```

Comment:
To be sure to correctly handle any invalid value (e.g. a non numeric string for instance).

```suggestion
        $max = (int)($task->fields['param'] ?? 0);
```